src/component/organism/box-slider-menu/box-slider-menu.tsx

```diff
@@ -2,12 +2,17 @@ import { FC, useContext } from "react";
 import TextCarousel from "../text-carousel/text-carousel";
 import MenuWrapper from "../../molecules/menu-wrapper/menu-wrapper";
 import { NavbarMenuContext } from "../../../context/navbar-menu-context";
+import { CategoriesModel } from "../../../api/api-get-categories";
 
-const BoxSliderMenu: FC = () => {
+interface PropsModel {
+  categories: CategoriesModel[];
+}
+
+const BoxSliderMenu: FC<PropsModel> = ({categories}) => {
     const { state, mediaQueryMatches } = useContext(NavbarMenuContext);
```

Comment:
`mediaQueryMatches` is not good name. It tells nothing about on which breakpoint it reacts.

Reply:
ok